content/docs/start/experiments/experiment-management.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Get Started: Experiment Management'
+description:
+  'Get started with managing experiments with DVC. Manage your experiments and
+  share them with others using the workflow that best suits your needs.'
+---
+
+# Get Started: Experiment Management
+
+After having compared all the experiments, ML Practitioners still need to agree
+on which one is the best and manage the remaining candidates.
+
+<abbr>DVC Experiments</abbr> are fully compatible with Git workflows, so you can
+manage the experiments using the workflow that best suits your needs.
+
+## Sharing
```

Comment:
Non-blocking suggestions:
1. Mention that this pushes everything needed to reproduce the experiment - the Git commit with all the experiment info does to your Git provider and any DVC-tracked data goes to your DVC remote storage.
2. Mention `dvc exp list` as a way to see what's been pushed, at least since we don't have Studio to show these yet.

Reply:
> Mention that this pushes everything needed to reproduce the experiment - the Git commit with all the experiment info does to your Git provider and any DVC-tracked data goes to your DVC remote storage.

I don't know if that is going too deep or not (as you pointed, this page already feels deep)

> Mention dvc exp list as a way to see what's been pushed, at least since we don't have Studio to show these yet.

Same as above. 

WDYT about adding a last section at the bottom of the page like "Go Further" or "What's next" linking to https://dvc.org/doc/user-guide/experiment-management/sharing-experiments ?

Reply:
> WDYT about adding a last section at the bottom of the page like "Go Further" or "What's next" linking to https://dvc.org/doc/user-guide/experiment-management/sharing-experiments ?

Yes, I think we should include something to show how to "go further/deeper" into a subject.